internal/runners/languages/install.go

```diff
@@ -4,24 +4,30 @@ import (
 	"strings"
 
 	"github.com/ActiveState/cli/internal/locale"
-	"github.com/ActiveState/cli/internal/output"
 	"github.com/ActiveState/cli/internal/primer"
+	"github.com/ActiveState/cli/internal/runbits/requirements"
 	"github.com/ActiveState/cli/pkg/platform/model"
 	"github.com/ActiveState/cli/pkg/project"
 )
 
 type Update struct {
-	out     output.Outputer
-	project *project.Project
+	prime primeable
```

Comment:
This feels inconsistent with how we usually write these. I'm pretty sure we only interpret primeable for the constructor for our other runners.